messages/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slackevents"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+// Response is of type APIGatewayProxyResponse since we're leveraging the
+// AWS Lambda Proxy Request functionality (default behavior)
+//
+// https://serverless.com/framework/docs/providers/aws/events/apigateway/#lambda-proxy-integration
+type Response events.APIGatewayProxyResponse
+
+var api = slack.New("TOKEN")
```

Comment:
Does that work ? 🤔

Reply:
Ideally, we'd need to read the secrets directly from AWS secrets/config manager, but maybe serverless already has nice support for secrets, @gjgd can help you with this I guess (we only need an environment variable)

Reply:
```suggestion
var api = slack.New("os.Getenv("SLACK_API_TOKEN")")
```